CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+Hello contributors, welcome to the OpenDAL Community! This document will guide you on how to participate in our community and enjoy your time here.
+
+## Taking Issues
+
+**OpenDAL's Issue Roundtrip: 14 days**
+
+It is highly recommended to request an assignment before beginning work on any issues to avoid conflicts with other community members.
+
+It is encouraged to post your work progress in the issues comment so that the community can stay informed about ongoing developments and offer any necessary assistance.
+
+It is possible that your assignment has been removed after 14 days of no updates. Please note that this is not a punishment or dissatisfaction from the community. We understand that life can be challenging, and things may not always go as planned.
+
+By removing the assignment, we create space for other community members to take over. Additionally, we do not frequently check the assignment due date, so please communicate any stale issues through the issue tracker.
+
+## Sending PRs
+
+**OpenDAL's PR Roundtrip: 24 hours**
+
+OpenDAL promises fast iteration (I mean very fast).
+
+We commit to reviewing your PRs within 24 hours, and often within 8 hours. In return, we expect you to respond to our review within 24 hours as well.
+
+If a PR receives no response for 24 hours, it will be closed. Please note that closing a PR does not mean rejection; we simply aim to keep our community iterating quickly by only tracking PRs awaiting review.
```

Comment:
Would you like to add the parts you always said? `The benefits for both of us why we close it`